README.md

```diff
@@ -1,3 +1,58 @@
 # rbac-config
 
-Repository for housing decoupled RBAC service policy configs
+==============================================================
+Predefined roles for Insights Role Based Access Control README
+==============================================================
+
+~~~~~
+About
+~~~~~
+
+Repository for housing decoupled RBAC service roles configs. These roles would be seeded into each
+tenant/account.
+
+Platform default roles are the roles associated with platform default group, which defines the default
+permissions for a principal.
+
+RBAC service repo: https://github.com/RedHatInsights/insights-rbac
+
+
+Development
+===========
+
+After you clone the RBAC service repo, replace contens in the insights-rbac/rbac/management/role/definitions folder 
+with the json files in the configs folder of rbac-config.
+
+
+Contributing
+=============
+
+Add new roles
+-------------
+Follow existing examples to add roles including name, description, system flag, acess with permissions.
+If you want the new role to be associated with platform default group (which defines the default permissions
+for principals in a tenant), you have to add the platform_default flag and set it as true.
+
+Set the version to 2 for the new role in order to trigger the seeding in the rbac service.
+
+Format of permissions
+^^^^^^^^^^^^^^^^^^^^^
+The permission should have the format of <application>:<resource>:<action>. There is no restrictions of the
+resource or action word, i.e. you could define any resource or action. It is up to App team to determine how to use the
+resource and action themself.
+
+Update roles
+-------------
+When you update the role, please change the version number of the role for the service to pick up new features.
+If you update the name of the role, which would generate a new role but keep the old one. Please reach out to 
+RBAC team to solve this.
+
+Delete roles
+-------------
+Currently, if you delete the role, it would still exist. Please reach out to RBAC team to delete them from database.
+
+
+Deployment
+==========
+When PR is merged to master/qa/prod branch, it would take up to 1 day to seed new roles in CI/QA/PROD. 
```

Comment:
```suggestion
When your PR is merged to master/qa/prod branch, it will take up to 1 day to seed new roles in CI/QA/PROD, as we have a daily task to update any new config. 
```